language/en-GB/en-GB.plg_system_httpheader.ini

```diff
@@ -0,0 +1,50 @@
+; HttpHeader Plugin
+; Copyright (C) 2017 - 2019 Tobias Zulauf All rights reserved.
+
+COM_PLUGINS_HSTS_FIELDSET_LABEL="Strict-Transport-Security (HSTS)"
+COM_PLUGINS_CSP_FIELDSET_LABEL="Content-Security-Policy (CSP)"
+PLG_SYSTEM_HTTPHEADER="System - HTTPHeaders"
+PLG_SYSTEM_HTTPHEADER_ADDITIONAL_HEADER="Define additional HTTP Header"
+PLG_SYSTEM_HTTPHEADER_ADDITIONAL_HEADER_DESC="Using this you can set also different values from the default ones and also additional headers when it is supported. The supported headers are: <br><ul><li>Strict-Transport-Security</li><li>Content-Security-Policy</li><li>Content-Security-Policy-Report-Only</li><li>X-Frame-Options</li><li>X-XSS-Protection</li><li>X-Content-Type-Options</li><li>Referrer-Policy</li><li>Expect-CT</li><li>Feature-Policy</li></ul>"
+PLG_SYSTEM_HTTPHEADER_ADDITIONAL_HEADER_KEY="HTTP Header"
+PLG_SYSTEM_HTTPHEADER_ADDITIONAL_HEADER_VALUE="HTTP Header Value"
+PLG_SYSTEM_HTTPHEADER_CONTENTSECURITYPOLICY="<a href='https://scotthelme.co.uk/content-security-policy-an-introduction' target='_blank'>Content Security Policy (CSP)</a>"
```

Comment:
as this is a target blank then it should have a rel=noopener noreferrer
yes I know scott's site will be safe but others may see this and copy it

Reply:
done that for all links and all languages now thanks.